### PR TITLE
Small clean-ups

### PR DESCRIPTION
**Pull request by edogaldo**

### CODE REVIEW

File: STM32F1/cores/maple/libmaple/exc.S [hunk 1]
```diff
@@ -41,7 +41,6 @@
 #    SP-->  r0
 
 .text
-.globl __exc_hardfault
```
Comment (edogaldo) [inline STM32F1/cores/maple/libmaple/exc.S]: Duplicate

**Comment by rogerclarkmelbourne** [comment]
Does this 

-#define CYCLES_PER_MICROSECOND  72

only affect one variant ?

**Comment by edogaldo** [comment]
Yes, it was the only one with 72 hardwired

**Comment by edogaldo** [comment]
Sorry no, I found others...

**Comment by BlackBrix** [comment]
I think there are many:
--> https://github.com/rogerclarkmelbourne/Arduino_STM32/search?utf8=%E2%9C%93&q=CYCLES_PER_MICROSECOND&type=

**Comment by edogaldo** [comment]
https://github.com/rogerclarkmelbourne/Arduino_STM32/blob/c2711e5f0ee35687e778a01cf75e0c498bd6e58d/STM32F1/libraries/SPI/src/SPI.cpp#L46-L49
Uhm.. guess this would match even if F_CPU=72000000...

**Comment by edogaldo** [comment]
Hello Roger, any interest in this PR?

**Comment by rogerclarkmelbourne** [comment]
Hi have done some of these manually

**Comment by edogaldo** [comment]
I've seen, not interested in other changes?

**Comment by rogerclarkmelbourne** [comment]
I'm going to close this, because the changes are not critical.

I may make some of these changes when I have some spare time, but thats not the case at the moment

**Pull request closed without merging**